src/panels/lovelace/common/process-config-entities.js

```diff
@@ -7,6 +7,10 @@ export default function processConfigEntities(entities) {
   }
 
   return entities.map((entityConf, index) => {
+    if (typeof entityConf === 'object' && !Array.isArray(entityConf) && entityConf.type) {
```

Comment:
If we do this let's change the name to reflect that this is a parser for entity row configs